cmd/estuary-shuttle/main.go

```diff
@@ -1944,9 +1939,10 @@ func (s *Shuttle) handleGetWantlist(c echo.Context) error {
 }
 
 type importDealBody struct {
-	Name       string   `json:"name"`
-	DealIDs    []uint64 `json:"dealIDs"`
-	Collection string   `json:"collection"`
+	util.ContentInCollection
```

Comment:
I'm curious how this works on Golang. Did you not name this field? What would its name be?

Reply:
yeah it isn't a named field at all, but an embedded type. it's a very Go-specific approach to common data fields and logic, since there is no such thing as a class hierarchy.

you can refer to fields on the embedded type by the type name.. more info here:
https://go.dev/doc/effective_go#embedding